g3/lbp_g3.c

```diff
@@ -1807,8 +1809,12 @@ lbp_g3_output(struct netif *netif, struct pbuf *p, struct lowpan6_link_addr *dst
   }
 
   LWIP_DEBUGF(LBP_G3_DEBUG, ("lbp_g3_output: lbp_g3_output: %02X%02X\n", dst->addr[0], dst->addr[1]));
-  return g3plc_output(netif, p, src, dst, security_level,
-                              ctx->pan_id, 0, ctx->active_key_index);
+
+  ret = g3plc_output(netif, p, src, dst, security_level, ctx->pan_id, 0, ctx->previous_key_index);
```

Comment:
Please add a comment explaining why and how `previous_key_index` is used (for new key activation). This may not be immediately obvious.